vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java

```diff
@@ -122,10 +122,19 @@ public void openPage(String pageURL)
      * </ul>
      * <p>
      * @param relativeURL A string value of the relative URL
+     * @deprecated <p>Please use step: "Then `$variable1` is $comparisonRule `$variable2`" instead.
+     * As "$variable1" please use expression "#{extractPathFromUrl($url)}"
+     * and as expression parameter "$url" please use dynamic variable "${current-page-url}"</p>
```

Comment:
see https://github.com/vividus-framework/vividus/pull/3847 on how to build deprecations